### PR TITLE
Add disable_komodo command to unsource komodo

### LINKED ISSUE #32: Add ($KOMODO_RELEASE) to the csh prompt
When you source Komodo in bash today you get the release as a prefix for your `PS1`, it would be nice to have the same in csh.

For bash, it is essentially

```bash
export PS1="(${KOMODO_RELEASE}) ${PS1}"
```

See the files [enable.in](https://github.com/equinor/komodo/blob/master/enable.in) (bash) and [enable.csh.in](https://github.com/equinor/komodo/blob/master/enable.csh.in) (csh).

### LINKED ISSUE #48: Create disable script
Create disable script such that the environment variables set by the enable script is reset to their original values. (Similar to how activate and deactivate works for virtualenv)

### PR DESCRIPTION
**Pull request by sondreso**
Resolves #48 
Resolves #32

### CODE REVIEW

File: enable.csh.in [hunk 1]
```diff
@@ -1,24 +1,57 @@
+changequote(<!,!>)
+
+alias fix_path '\\
+    setenv PATH `echo "$PATH" | sed -e "s|$_KOMODO_PATH_EXTENSION\:*||g"` && unset _KOMODO_PATH_EXTENSION;\\
+    rehash;\\
```
Comment (ManInFez) [inline enable.csh.in]: rehash only once at the end fo disable_komodo

**Comment by markusdregi** [comment]
This removes the `komodo` part only from the environment right? Are we sure we want to go down the road of special behavior? If your `PATH` is `/not/komodo/compliant/bin:/the/official/komodo/install/bin:/usr/bin` then as I understand it `deactivate` will give you the path `/not/komodo/compliant/bin:/usr/bin`. And then, if we source again, as always you get: `/the/official/komodo/install/bin:/not/komodo/compliant/bin:/usr/bin`.

Hence, deactivating and activating environments will not give you something that resembles your old environment even if we try. I suggest we lean on the previously given instruction that our users are not to edit their environment after they source komodo and do a plain deactivate in the same style as `venv` etc.

**Comment by sondreso** [comment]
@ManInFez :arrow_up:

**Comment by oysteoh** [comment]
Closes #32

**Comment by sondreso** [comment]
This has been updated with a bug fix, still good?

**Comment by markusdregi** [comment]
You have conflicts. Please resolve them ;)

**Comment by markusdregi** [comment]
Looks good 👍

**Pull request merged**